README.md

```diff
@@ -95,36 +112,106 @@ Configure in `*.csproj` - project settings:
 </Project>
 ```
 
+## Analyzers
```

Comment:
Nowa sekcja

Reply:
Tutaj ten tekst to tak trochę byle jak, Amadeusz dobrze jakby na to sporzał czy to jest poprawny "powód"/"rekomendacja"